src/App.js

```diff
@@ -11,7 +12,21 @@ const BOOK_QUERY = gql`
 function App() {
   const { data, loading, error } = useQuery(BOOK_QUERY);
   console.log(data);
-  return <div>Hello World</div>;
+  return (
+    <div>
+      <Router>
+        
+        <Switch>
+          <Route path="/register">
+            register page
+          </Route>
+          <Route path="/signin">
+            sign in page
+          </Route>
+        </Switch>
+      </Router>
+    </div>
+  );
```

Comment:
This is a nit pick... You can most likely add fragments to this as we don't need the explicit `div` tags. 

```suggestion
  return (
    <>
      <Router>
        <Switch>
          <Route path="/register">
            register page
          </Route>
          <Route path="/signin">
            sign in page
          </Route>
        </Switch>
      </Router>
    </>
  );
```